### DISCUSSION TITLE
Flipper Zero script ! Need Help.

### DISCUSSION THREAD

**Discussion by Fapp3r3Lit3** [community/community#79192] [Programming Help]
I was in the middle of creating g this script when pc took a dump. Lost some files and other work for it and tried to piece what I had saved together . Can't remember too many contributors and finishing touches 

Help me learn. Help me create this. Thanks in advance

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Fapp3r3Lit3! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, you are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Coco19800** [reply]
Ty for the pointers
